recipes/docker-image-guide.md

```diff
@@ -187,15 +189,15 @@ ENTRYPOINT ["dumb-init", "--"]
 FROM node:20.9-bookworm@sha256:3c48678afb1ae5ca5931bd154d8c1a92a4783555331b535bbd7e0822f9ca8603 AS build
 WORKDIR /usr/src/app
 COPY package*.json .
-RUN npm ci
+RUN npm ci --omit=dev
```

Comment:
In this step, we may want to keep dev dependencies as this is a build step that probably requires some build tools that are available as dev dependencies.

Reply:
You are correct. Building the frontend should either require dev dependencies to be installed in Docker or move build lib (vite, webpack, ...) to prod dependencies.

Reply:
But also, we should set NODE_ENV to production in this step, import .env, set secrets etc. What I'm saying is, we should add more use cases and details in the future.